buildpack-deps/content.md

```diff
@@ -12,3 +12,26 @@ knowing beforehand that `ssl.h` is required to build a dependent module.
 # How to use this image
 
 This stack is designed to be the foundation of a language-stack image.
+
+## What's included?
+
+The main tags of this image are the full batteries-included approach.  With
+them, any arbitrary `gem install` should work.  For some language stacks, that
```

Comment:
This should be "the majority of", not "any".  If we go as broad as "any", we're wide open for including the entire Debian archive. :+1: